managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedgerConfig.java

```diff
@@ -51,6 +51,7 @@
     private double throttleMarkDelete = 0;
     private long retentionTimeMs = 0;
     private long retentionSizeInMB = 0;
+    private boolean skipNonRecoverableLedger;
```

Comment:
what about naming it `automaticallySkipNonRecoverableData`?

Reply:
or abbreviated into `auto`

Reply:
sure, I will change it.